kindlemint/data/private_data_pipeline.py

```
@@ -0,0 +1,985 @@
+            DataCategory.BEHAVIORAL: [
+                r".*click.*", r".*view.*", r".*visit.*", r".*search.*",
+                r".*preference.*", r".*rating.*", r".*bookmark.*"
+            ],
+            DataCategory.TRANSACTIONAL: [
+                r".*order.*", r".*payment.*", r".*transaction.*",
+                r".*purchase.*", r".*invoice.*", r".*refund.*"
+            ]
+        }
+        
+        # Categorize each field
+        for field, value in raw_data.items():
```

Ruff (0.11.9)

Import `field` from line 12 shadowed by loop variable

(F402)